x-pack/legacy/plugins/spaces/public/views/management/components/copy_saved_objects_to_space/copy_status_indicator.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiLoadingSpinner, EuiText, EuiIconTip } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { SavedObjectsImportMissingReferencesError } from 'src/core/server';
+import {
+  SummarizedCopyToSpaceResult,
+  SummarizedSavedObjectResult,
+} from '../../../../lib/copy_saved_objects_to_space';
+
+interface Props {
+  summarizedCopyResult: SummarizedCopyToSpaceResult;
+  object: { type: string; id: string };
+  overwritePending: boolean;
+  conflictResolutionInProgress: boolean;
+}
+
+export const CopyStatusIndicator = (props: Props) => {
+  const { summarizedCopyResult, conflictResolutionInProgress } = props;
+  if (summarizedCopyResult.processing || conflictResolutionInProgress) {
+    return <EuiLoadingSpinner />;
+  }
+
+  const objectResult = summarizedCopyResult.objects.find(
+    o => o.type === props.object!.type && o.id === props.object!.id
+  ) as SummarizedSavedObjectResult;
+
+  const successful =
+    !objectResult.hasUnresolvableErrors &&
+    (objectResult.conflicts.length === 0 || props.overwritePending === true);
+  const successColor = props.overwritePending ? 'warning' : 'success';
+  const hasConflicts = objectResult.conflicts.length > 0;
+  const hasUnresolvableErrors = objectResult.hasUnresolvableErrors;
+  const hasMissingRefs = objectResult.missingReferences.length > 0;
+
+  if (successful) {
+    const message = props.overwritePending ? (
+      <FormattedMessage
+        id="xpack.spaces.management.copyToSpace.copyStatus.pendingOverwriteMessage"
+        defaultMessage="Saved object will be overwritten. Click 'Skip' to cancel this operation."
+      />
+    ) : (
+      <FormattedMessage
+        id="xpack.spaces.management.copyToSpace.copyStatus.successMessage"
+        defaultMessage="Saved object copied successfully."
+      />
+    );
+    return <EuiIconTip type={'check'} color={successColor} content={message} />;
+  }
+  if (hasMissingRefs) {
```

Comment:
Were you able to test this functionality somehow? I get a 400 whenever trying...

Reply:
See [comment above](https://github.com/elastic/kibana/pull/39002#discussion_r316951202), I think we're talking about two different scenarios